Sources/StorageCommitmentScpCallback.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * Python plugin for Orthanc
+ * Copyright (C) 2020-2022 Osimis S.A., Belgium
+ * Copyright (C) 2021-2022 Sebastien Jodogne, ICTEAM UCLouvain, Belgium
+ * Copyright (C) 2022 Johnson Sun
```

Comment:
Don't think we can keep this once you've signed the CLA and once we merge the code :-(

Reply:
No problem. I removed my name in both the header and C file (0d7a9306415aa860b63b70a826f9902dd1cc1944). (I don't mind at all)

I thought the names above are meant to track who's responsible for bugs in the code, so I followed suit.